src/backend/distributed/commands/cascade_table_operation_for_connected_relations.c

```diff
@@ -426,3 +428,54 @@ ExecuteAndLogDDLCommand(const char *commandString)
 	CitusProcessUtility(parseTree, commandString, PROCESS_UTILITY_TOPLEVEL,
 						NULL, None_Receiver, NULL);
 }
+
+
+/*
+ * ExecuteAndLogDDLCommandList takes a list of foreign key creation ddl commands
+ * and calls ExecuteAndLogForeignKeyCreateCommand function for each of them.
+ */
+void
+ExecuteAndLogForeignKeyCreateCommandList(List *ddlCommandList, bool skip_validation)
```

Comment:
I don't quite get the AndLog in these function names. At least, I would not consider DEBUG4 logging.

Reply:
good point